test_sherlock_anagrams.py

```diff
@@ -0,0 +1,31 @@
+
+
+def sherlock_anagrams(word):
+    """
+      Time: O(n^2 * nlogn) + O(n)
+      Space: O(n^2)
+    """
+    size = len(word)
+    occ = {}
+
+    # calculate all substrings and yours occurencies
+    for i in range(size):
+        sub_string = ''
+        for j in range(i, n):
+            sub_string = ''.join(sorted(sub_string + word[j]))
+            occ[sb] = occ.get(sub_string, 0)
```

Comment:
@matheusfrancisco just a small typo here on the key name

Reply:
nice, thks  :heart: